examples/next-prisma-starter/jest.config.ts

```diff
@@ -1,4 +1,24 @@
 import type { Config } from '@jest/types';
```

Comment:
Maybe we should switch this to use [next/jest](https://nextjs.org/docs/testing#setting-up-jest-with-the-rust-compiler) so it uses the swc compiler.

Reply:
> "next/jest" is currently experimental. https://nextjs.org/docs/messages/experimental-jest-transformer

skipped it because above warning when doing it